src/main/java/com/example/auctionapp/Util/EnumUtility.java

```diff
@@ -0,0 +1,23 @@
+package com.example.auctionapp.Util;
+
+import com.example.auctionapp.enumeration.ColorEnum;
+import com.example.auctionapp.enumeration.SizeEnum;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class EnumUtility {
+
+    public static List<String> getColors() {
+        return Arrays.stream(ColorEnum.values()).map(name -> {
+            return name.label;
+        }).collect(Collectors.toList());
+    }
+
+    public static List<String> getSizes() {
+        return Arrays.stream(SizeEnum.values()).map(name -> {
+            return name.label;
+        }).collect(Collectors.toList());
+    }
```

Comment:
Now that we have this simple way to get the values of enums, these methods can go into the enum itself. As enums by definition are static, there is no issue to put these methods in the enum itsself